### ISSUE TITLE
How can I create HTML output in different speaking language (English and German)?

### ISSUE THREAD

**Issue by nefissakhd** [bquast/multilang#1]
I'm currently working on a RMarkdown document in English, I want to knit my document in German also, so I wrote the translated rmarkdown documents in German, now I have a similar structure to this :

    content
    	├── about.en.md
    	├── about.de.md
    	└── exp.en.md
    	├── exp.de.md
    	├── index.md
    	└── index.en.md
    	
	
this is the code in my header where I have the hyperlink 

    > <div id="pagetitle">
>     title
>     </div>
>     
>     <div class="lang">
>       <ul>
>         
>           <li><a class="de" href="index-de.html">
>             De </a></li>
>         
>           <li><a class="en" href="index-en.html">En</a></li>
>         
>       </ul>
>     </div>

this is my _site.yml code: 

   >  name: "title"
>     output:
>       html_document:
>         output_dir: "."
>       include:
>           before_body: header.html
>         css: style.css
        

   

how can i use your projec to do this? thank you

**Comment by bquast** [comment]
Hi @nefissakhd tbh this was just a small experiment that I made a while ago, I was not aware people were using this, let me look at this tomorrow (I have to install it again myself I guess)

**Comment by bquast** [comment]
@nefissakhd did you look at the Vignette? (https://github.com/bquast/multilang/blob/master/vignettes/Candide.Rmd) Does that work on your machine?

**Comment by nefissakhd** [comment]
yes I've had a look, but the thing is that when having multiple files where
the navigation is in external html file ( tab) it doesn't really work...

**Comment by bquast** [comment]
Let me check tomorrow (hopefully)

**Comment by bquast** [comment]
Otherwise on the eastern weekend

**Comment by bquast** [comment]
I would combine with the [readtext](https://cran.r-project.org/package=readtext) packager ([vignette](https://cran.r-project.org/web/packages/readtext/vignettes/readtext_vignette.html)). 

The [inserttext function](https://github.com/bquast/multilang/blob/master/R/multilang.R) is really simple, you could simply copy that (note the argument the YAML preamble: language: English)
